src/dynimage.rs

```diff
@@ -913,14 +925,23 @@ pub fn load<R: BufRead + Seek>(r: R, format: ImageFormat) -> ImageResult<Dynamic
     }
 }
 
+fn load_guess<R: Read + Seek>(mut r: R) -> ImageResult<DynamicImage> {
+    let mut buffer: [u8; 16] = [0u8; 16];
+    let cur = r.seek(SeekFrom::Current(0))?;
+    r.read_exact(&mut buffer[..])?;
+    r.seek(SeekFrom::Start(cur))?;
+    load(r, guess_format(&buffer)?)
+}
+
 static MAGIC_BYTES: [(&'static [u8], ImageFormat); 17] = [
     (b"\x89PNG\r\n\x1a\n", ImageFormat::PNG),
     (&[0xff, 0xd8, 0xff], ImageFormat::JPEG),
     (b"GIF89a", ImageFormat::GIF),
     (b"GIF87a", ImageFormat::GIF),
     (b"RIFF", ImageFormat::WEBP), // TODO: better magic byte detection, see https://github.com/image-rs/image/issues/660
-    (b"MM.*", ImageFormat::TIFF),
-    (b"II*.", ImageFormat::TIFF),
+    (b"WEBP", ImageFormat::WEBP),
```

Comment:
I think this line shouldn't be here?

Reply:
rebase went wrong, fixed.

thanks for noticing!